app/src/main/java/org/fossasia/susi/ai/data/contract/IUtilModel.kt

```diff
@@ -0,0 +1,24 @@
+package org.fossasia.susi.ai.data.contract
+
+import org.fossasia.susi.ai.rest.responses.susi.LoginResponse
+import retrofit2.Response
+
+/**
+ * Interface for Util Model
+ *
+ * Created by chiragw15 on 10/7/17.
+ */
+interface IUtilModel {
```

Comment:
I think IUtilModel will be different for login, signup and forgotpassword? I should be named accordingly. Like ILoginUtilModel.